react/features/overlay/components/ReloadButton.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+
+import { translate } from '../../base/i18n';
+
+import { reconnectNow } from '../functions';
```

Comment:
The Formatting section of doc/codying-style.md suggests it in a vague way. So the place to document it would be there.